Jenkinsfile

```diff
@@ -66,13 +82,19 @@ node('ios-osx') {
                 }
             }
         }
+
         stage('Upload Results') {
             archiveArtifacts allowEmptyArchive: true, artifacts: 'external/autobots/*.log'
+
             junit "external/autobots/test-reports/*.xml"
-            zip archive: true, dir: 'external/autobots/screenshots', glob: '', zipFile: 'external/autobots/screenshots.zip'
+
+            try {
+                zip archive: true, dir: 'external/autobots/screenshots', glob: '', zipFile: 'external/autobots/screenshots.zip'
+            } catch (e) {
+                // no screenshots, no problem
+            }
```

Comment:
This would no longer fail the Build if there was something wrong with the Run Tests Stage. Is there something we can do in the Run Tests Stage to Fail the build process ?

Reply:
good question ;-)

Reply:
I have a check to Fail the Jenkins Stage if anything goes wrong during the "python testRunner.py" step.

But the PR still fails due to the above comment from @sammacbeth .